examples/platformio/src/main.cpp

```diff
@@ -51,6 +56,8 @@ void setup() {
     // sensors.init(sensors.Auto);                        // Auto detection of PM sensors (Honeywell, Plantower, Panasonic)
     // sensors.init(sensors.Panasonic);                   // Force detection to Panasonic sensor
     // sensors.init(sensors.Sensirion);                   // Force detection to Sensirion sensor
+    // sensors.init(sensors.Mhz19);                       // Force detection to Mhz14 or Mhz19 CO2 sensor 
+    // sensors.init(sensors.CM1006);                      // Force detection to CM1106 CO2 sensor
```

Comment:
or these comments only is for show this possibility?

Reply:
yes